package-system/AWSNativeSDK/README.md

```diff
@@ -0,0 +1,101 @@
+# Build New AWS Service Target in AWSNativeSDK
+First check package directories at [aws/aws-sdk-cpp](https://github.com/aws/aws-sdk-cpp) on GitHub. 
```

Comment:
Maybe, "Find and identify the service package required from the package directory at ..."?